core/src/main/java/org/elasticsearch/search/internal/InternalSearchHit.java

```diff
@@ -438,7 +438,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 if (field.values().isEmpty()) {
                     continue;
                 }
-                if (field.isMetadataField()) {
+                // "_all" should be treated as an ordinary field, issue 13178
+                if (field.isMetadataField() && !field.name().equals("_all")) {
```

Comment:
I think you could use the constant [Metadata.ALL](https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java#L85) here.